sdk/search/azure-search-documents/samples/async_samples/sample_indexers_operations_async.py

```diff
@@ -0,0 +1,125 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_indexer_operations_async.py
+DESCRIPTION:
+    This sample demonstrates how to get, create, update, or delete a Indexer.
+USAGE:
+    python sample_indexer_operations_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_SEARCH_SERVICE_ENDPOINT - the endpoint of your Azure Cognitive Search service
+    2) AZURE_SEARCH_API_KEY - your search API key
+"""
+
+import asyncio
+import os
+
+service_endpoint = os.getenv("AZURE_SEARCH_SERVICE_ENDPOINT")
+key = os.getenv("AZURE_SEARCH_API_KEY")
+connection_string = os.getenv("AZURE_STORAGE_CONNECTION_STRING")
+
+from azure.core.credentials import AzureKeyCredential
+from azure.search.documents import (
+    DataSource, DataContainer, DataSourceCredentials, Index, Indexer, SimpleField, edm
+)
+from azure.search.documents.aio import SearchServiceClient
+
+service_client = SearchServiceClient(service_endpoint, AzureKeyCredential(key))
+indexers_client = service_client.get_indexers_client()
+
+async def create_indexer():
+    # create an index
+    index_name = "hotels"
+    fields = [
+        SimpleField(name="hotelId", type=edm.String, key=True),
+        SimpleField(name="baseRate", type=edm.Double)
+    ]
+    index = Index(name=index_name, fields=fields)
+    ind_client = service_client.get_indexes_client()
```

Comment:
Eventually, the method name should comply with the guidelines, i.e. that the model name is part of not only the client name ({Model}Client), but the method is get_{Model}Client (granted, in snake case idiomatic of python). Might consider doing before the UX study so, if people say "that's too long", we can take that back to archboard.